EIPS/eip-5189.md

```diff
@@ -0,0 +1,248 @@
+---
+eip: <to be defined>
+title: Account Abstraction via Endorsed Operations
+description: An account abstraction proposal that avoids consensus-layer protocol changes while maintaining compatibility with existing smart contract wallets.
+author: Agustín Aguilar (@agusx1211), Philippe Castonguay (@phabc).
+discussions-to: https://ethereum-magicians.org/t/erc-account-abstraction-via-endorsed-operations/9799
+type: Standards Track
+category: ERC
+status: Draft
+created: 2022-06-29
```

Comment:
```suggestion
created: 2022-06-29
requires: 4337
```

Reply:
Then it shouldn't be mentioned.  Standards documents are not the right place to debate/compare/contrast specific proposals.  There is value in briefly describing specific alternatives that were considered in the Rationale section, but you shouldn't be referring to them by EIP number since that creates a dependency that you almost certainly don't want.